xml/repeated-content-decl.ent

```diff
@@ -97,7 +97,12 @@ Before you start deploying &productname;, review the following documents:
 '<para xmlns="http://docbook.org/ns/docbook">
 The following tables
 list the minimum DNS requirements to run &productname;, using
-   <literal>example.com</literal> as the example domain:
+   <literal>example.com</literal> as the example domain. Your DNS management is
+   platform-dependent, for example Microsoft Azure assigns IP addresses to your
+   services, which you will map to your domains. Amazon EKS assigns hostnames,
```

Comment:
Should specify it's for A records since in the EKS sentence it specifies CNAMES

Reply:
Good catches :+1: